pkg/mic/mic_test.go

```diff
@@ -20,13 +20,17 @@ import (
 	cp "github.com/Azure/aad-pod-identity/pkg/cloudprovider"
 	api "k8s.io/api/core/v1"
 	corev1 "k8s.io/api/core/v1"
-	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/cache"
 	"k8s.io/klog"
 )
 
+var (
+	testResourceID = "/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/rg/providers/Microsoft.ManagedIdentity/userAssignedIdentities/identity1"
```

Comment:
Could we add set of negative test cases with wrong format of resource id ?

Reply:
The negative test case is already in here - https://github.com/Azure/aad-pod-identity/pull/548/files#diff-33233ef23f7b38ee10add9aed587cd05R770-R775.